app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
```

Comment:
The `User` model has been added with a `settings` method that returns an `OpenStruct`. Using `OpenStruct` can be slower than using a `Hash` or a custom class and may not be necessary. Consider refactoring to use a more performant data structure if performance is a concern.